### PR TITLE
React-app setup and linters

### PR DESCRIPTION
**Pull request by chepkok3**
- [ ] Set up a React app.
- [ ] Set up react linters.
- [ ] Added descriptive README.md file.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
```
Comment (DammyShittu) [inline src/App.js:8]: - [ ] When I run your app I get this error message which prevents the app from running. Kindly fix this. You need to add back the deleted `src/logo.svg` file or remove its use in this file.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @chepkok3,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (chepkok3) [reply src/App.js:8]: @DammyShittu Alright. I got that.

Comment (chepkok3) [reply src/App.js:8]: @DammyShittu Done.

Comment (Whoistolu) [reply src/App.js:8]: - Well resolved @chepkok3 
Screenshot 2023-02-20 at 19 34 06

Comment (chepkok3) [reply src/App.js:8]: @Whoistolu 👍🏿

File: README.md [hunk 2]
```diff
@@ -1,70 +1,161 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```
Comment (Whoistolu) [inline README.md]: - [ ] [OPTIONAL] Kindly ensure that this project is properly licensed. Currently, the License is broken. The License helps us indicate that you have ownership of the project.
Screenshot 2023-02-20 at 19 35 16

File: README.md [hunk 3]
```diff
@@ -1,70 +1,161 @@
...
+- GitHub: [@chepkok3](https://github.com/chepkok3)
+- Twitter: [@chepkok3](https://twitter.com/home)
+- LinkedIn: [@chepkok3](https://www.linkedin.com/in/kibor-stanley-350b8a123/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **React components.**
+
```
Comment (Whoistolu) [inline README.md]: - [ ] [OPTIONAL] Kindly ensure to add at least two features you think this app will have in the future. 'React components'  as you have stated is not a feature.

**Review by Whoistolu — APPROVED** [review]
### Hi @chepkok3 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (chepkok3) [reply README.md]: @Whoistolu Noted.

**Pull request merged**